server/api/submissions.js

```
@@ -57,19 +57,13 @@ router.post('/:teamID/create', (req, res) => {
   // });
 });
 
-// Update isSubmitted to true
-router.patch('/:teamID/:submissionID/submit', async (req, res) => {
-  await Submission.findByIdAndUpdate(req.params.submissionID, {
-    isSubmitted: true,
+// update a submission with new info
+router.patch('/:teamID/:submissionID', async (req, res) => {
+  await Submission.findOneAndUpdate(req.params.submissionID, req.body, {
+    new: true,
+    upsert: true,
```

no need for the upsert, because we already have a post route to create a new submission

upsert basically means to _update_ a document, and if the document doesnt exist, then _insert_ it, hence _upsert_. In our case, if we dont find a document then we can just throw an error rather than inserting it